.buildkite/pipeline.yml

```diff
@@ -325,23 +349,23 @@
       --test_env=CONDA_SHLVL
       --test_env=CONDA_PREFIX
       --test_env=CONDA_DEFAULT_ENV
-      --test_env=RAY_gcs_grpc_based_pubsub=true
+      --test_env=RAY_gcs_grpc_based_pubsub=1
```

Comment:
why change this? `=true` should work I think.

Reply:
Yes `=true` also works but `=1` just seems cleaner to me. But it makes no big difference..